### PR TITLE
docs(apple-pay): Add Apple Pay documentation to connectors [TA-465]

### PR DESCRIPTION
**Pull request by cbetta**
Adds a new guide that explains how to set up and use Apple Pay.

### CODE REVIEW

File: docs/connectors/apple-pay.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+---
+title: Apple Pay
+sidebar_label: Apple Pay
+slug: /connectors/apple-pay
+---
+
+Apple Pay is a digital wallet service by Apple that allows buyers to use
+previously stored payment methods stored on their Apple device. It is supported
```
Comment (gryevns) [inline docs/connectors/apple-pay.md:8]:
```suggestion
previously stored payment methods on their Apple device. It is supported
```

File: docs/connectors/apple-pay.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+The merchant name and URL of your company need to be configured for Apple Pay.
+These are not checked but will be shown in some circumstances to the buyer
+during checkout with Apple Pay.
+
+### Domain names & verification
+
+You will need to register the domain of every site you will want to provide
+the Gr4vy Embed checkout on when setting up Apple Pay. As part of this process,
+Apple will very the presence of a **Domain Association File** on each of the
```
Comment (gryevns) [inline docs/connectors/apple-pay.md:33]:
```suggestion
Apple will verify the presence of a **Domain Association File** on each of the
```

File: docs/connectors/apple-pay.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+In short, this prevents Apple Pay from being incorporated into sites that
+violates any law, sells tobacco, firearms, and other prohibited items. Please
+check the
+[guidelines](https://developer.apple.com/apple-pay/acceptable-use-guidelines-for-websites/)
+for more details.
+
+## Usage & Testing
+
+To use Apple Pay it requires the user to be using Safari on a compatible
+devices. Apple Pay can be set up on a
```
Comment (gryevns) [inline docs/connectors/apple-pay.md:60]:
```suggestion
device. Apple Pay can be set up on a
```

**Review by gryevns — APPROVED** [review]

File: docs/connectors/apple-pay.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
+---
+title: Apple Pay
+sidebar_label: Apple Pay
+slug: /connectors/apple-pay
+---
+
+Apple Pay is a digital wallet service by Apple that allows buyers to use
+previously stored payment methods on their Apple device. It is supported
+on the iPhone and Mac. It is not available on any device that is not made and
```
[the remark this answers is not in the document]
Comment (cbetta) [reply docs/connectors/apple-pay.md:9]: This is how it's described on wikipedia and I think on one of their own pages. I think the key issue is that it won't work on a hackintosh.

**Comment by douglaseggleton** [comment]
This is not a connector - Can we we rename the `Connectors` section to `Connections`

**Comment by cbetta** [comment]
> This is not a connector - Can we we rename the `Connectors` section to `Connections` Oh wait this might just be the slug/path being wrong. <- In which case can update separately if you want.

Yeah path had been `connectors` for a while. I can look at updating that with a remap.

**Review by douglaseggleton — APPROVED** [review]
🚀

**Pull request merged**